modules/cloud-config-container/simple-nva/main.tf

```diff
@@ -15,40 +15,85 @@
  */
 
 locals {
```

Comment:
we should first put in alphabetical order all the locals prefixed with _
Then in alphabetical order all the other locals

Reply:
Now it should be fine, first locals with _ in alphabetical order and then other locals!